contents/blog/open-source-business-models.md

```diff
@@ -0,0 +1,68 @@
+
+Let’s now look at how we’ve managed to turn a buck with PostHog over the past few months — plus 3 other business models to explore for your own product.	
+
+## Option #1: Support
+Support involves helping large companies deploy, use, maintain, and upgrade your software. These companies usually lack the time, expertise, or inclination to do all that by themselves, and as the developer, you’re in a better position to tailor the deployment to their specific situation, deploy updates as needed, and extract a lot of value.
+
+The downside is that support-based open source business models tend to return lower margins. This is fine if you’re bootstrapping, but trying to raise VC money becomes a [schlep](https://www.google.com/search?q=schlep) as VC firms generally prefer high-margin startups.
+
+At PH, we started with the support model before running into one problem: developers typically want to wrangle your software themselves before paying for help or maintenance. 
+
+<iframe src="https://giphy.com/embed/pVsn5LJEgMKxa" width="480" height="269" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/xbox-ps4-tells-pVsn5LJEgMKxa">via GIPHY</a></p>
+
+This makes it a hard sell, but you can overcome this if you have more ‘clout’ by being a large project with hundreds of contributions and thousands of commits. [RedHat](https://www.redhat.com/en) is a good example of a support-based open source company.
+
+## Option #2: Hosted vs. Self-Hosted
+With this option, you offer users the option to either host your product on their own servers or host it on yours. The value prop is similar to that of the support model: companies don’t want the hassle of hosting your product themselves and are willing to pay you for it. The benefit here is that you only have to maintain one version of your product, which makes it easier and more financially feasible to plan your roadmap and hire a smaller team.
+
+The downside of the hosted model is that it’s not very defensible. You’ll likely face competition from hosting providers who simply host your free version and charge their users for extra value-adds, cutting you out of the deal completely. 
+
+<iframe src="https://giphy.com/embed/j4ksBQ70a3ECh2VObN" width="480" height="270" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/lastmanstandingfox-foxtv-last-man-standing-j4ksBQ70a3ECh2VObN">via GIPHY</a></p>
+
+The hosting business model is also much harder to pull off without a solid, popular product. [OpenStack](https://www.openstack.org/) is an example of a product that lets you self-host or host with a 3rd party provider.
```

Comment:
This example confused me  - OpenStack doesn't have pricing for a hosted version on their website.

The classic examples here are MongoDB or Sentry.

Reply:
Amended